src/main/java/org/prebid/server/proto/openrtb/ext/request/lunamedia/ExtImpLunamedia.java

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.proto.openrtb.ext.request.lunamedia;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+/**
+ * Defines the contract for bidRequest.imp[i].ext.lunamedia
+ */
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtImpLunamedia {
+
+    @JsonProperty("pubid")
+    String pubId;
```

Comment:
It not strongly required to name the field camel case,  `pubid` if fine. So no annotation will be needed.